readthedocs/templates/mkdocs/readthedocs/versions.html

```diff
@@ -0,0 +1,8 @@
+  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
```

Comment:
This is the only customisation needed in ReadTheDocs, so I made it easier but rolling a [small MkDocs release](http://www.mkdocs.org/about/release-notes/#version-0132-2015-05-30). See: https://github.com/mkdocs/mkdocs/pull/572